### ISSUE TITLE
Webhook Events

### ISSUE THREAD

**Issue by jetstream0** [repldex/Repldex#201]
Eg: entry edited request to webhook sent

**Comment by selectthegang** [comment]
Good idea, but would be hard without a server. Am setting this to In Review so mat can see if it's a good idea or not
